packages/veto-frontend/src/pages/Welcome/index.tsx

```diff
@@ -1,26 +1,19 @@
-import React, { useState } from 'react'
+import React from 'react'
 import { useInput } from '../../hooks/useInput'
 import { navigate } from '@reach/router'
 import { Flex, Text, Input, Button } from '@veriphi/veto-ui'
 import { mockPassword } from './tempdata.json'
-import axios from 'axios'
 
 export default () => {
   const password = useInput('')
 
-  const handleOnLogin = () => {
+  const handleOnLogin = (e) => {
```

Comment:
Please change to `event` to reduce cognitive burden (need to look at its usage to know its an event)